.github/workflows/ci.yml

```
@@ -436,6 +436,36 @@ jobs:
 
+      - name: 'Add a link to test report in summary'
+        if: ${{ strategy.job-index == 0 }} # only add the link once
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const BASE_REPORT_URL = 'https://woocommerce.github.io/woocommerce-test-reports/#/';
+            function getReportUrl() {
+                if (context.eventName === 'pull_request') {
+                    return `${BASE_REPORT_URL}pr/${context.issue.number}`;
+                }
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)